data/citensg-1.0/styles/default_point.sld

```diff
@@ -12,7 +12,7 @@
 		<UserStyle>
 		    <!-- they have names, titles and abstracts -->
 		  
-			<Title>A boring default style</Title>
+			<Title>A simple default style</Title>
```

Comment:
Title goes in legend, so please short descriptive text. "Red point" for example. (the abstract needs fixing too, the point is not purple). Can you fix the others accordingly too?
To check if the description matches the appearance use the map preview in the style editor of GeoServer, you'll get the legend overlayed with the map.

Reply:
understood, will re check all.